Program.cs

```diff
@@ -0,0 +1,215 @@
+        {
+            Console.Write("\nEnter class name: ");
+            var name = Console.ReadLine();
+            if (!string.IsNullOrEmpty(name)) 
+                _firstSchool.AddClass(name.ToString());
+            CleanConsoleBuffer();
+            Start();
+        }
+
+        private static void AddMemberToSchool()
+        {
+            _firstSchool.AddMember(GetMemberDataFromConsole());
+            Start();
+        }
+        
+        private static void AddMemberToClass()
+        {
+            if(CheckSomeClassExist())
+            {
+                var className = ShowAndChooseClassesAvailable();
+                if (!string.IsNullOrEmpty(className))
+                {
+                    var _class = _firstSchool.FindAndGetClass(className);
+                    Console.WriteLine("Found class: " + _class.Name);
+                    _class.AddMember(GetMemberDataFromConsole());
+                }
+            }
+            Start();
+        }
+        
+        private static void ShowSpecificClassMembers()
+        {
+            if (CheckSomeClassExist())
+            {
+                var className = ShowAndChooseClassesAvailable();
+                if (!string.IsNullOrEmpty(className))
+                {
+                    var _class = _firstSchool.FindAndGetClass(className);
+                    Console.WriteLine("Found class: " + _class.Name);
+                    _class.ShowMembers();
+                }
+            }
+            Start();
+        }
+
+        private static void LeavingSchool()
+        {
+            Console.WriteLine("...in school building:");
+            _firstSchool.ShowMembers();
+            Console.Write("\nType part of name member to leave: ");
+            CleanConsoleBuffer();
+            var name = Console.ReadLine();
+            _firstSchool.LeavingSchool(name);
+            CleanConsoleBuffer();
+            Start();
+        }
+
+        private static void ShowMembersSortedByID()
+        {
+            _firstSchool.SortByID(_firstSchool._humans);
+            ShowPeopleInSchool();
+        }
+
+        private static void ShowMembersSortedByName()
+        {
+            _firstSchool.SortByName(_firstSchool._humans);
+            ShowPeopleInSchool();
+        }
+
+        private static Human GetMemberDataFromConsole()
+        {
+            Console.WriteLine("Adding member...");
+            Console.Write("Enter name: ");
+            var name = Console.ReadLine();
+            CleanConsoleBuffer();
+            Console.Write("Enter lastname: ");
+            CleanConsoleBuffer();
+            var lastName = Console.ReadLine();
+            CleanConsoleBuffer();
+            Console.Write("Enter ID: ");
+            var ID = Convert.ToInt32(Console.ReadLine());
+            CleanConsoleBuffer();
+            
+            return new Human(name, lastName, ID);
+        }
+        
+        private static string ShowAndChooseClassesAvailable()
+        {
```

Comment:
Some of these methods work for the firstSchool only.

Reply:
ok